src/components/Home/data/brands.js

```diff
@@ -0,0 +1,17 @@
+import featImg1 from "./../../../assets/brands/reymond.png";
+import featImg2 from "./../../../assets/brands/spykar.png";
+import featImg3 from "./../../../assets/brands/levis.png";
```

Comment:
incase you don't know: you can directly use image paths just as we use in HTML (assets must be in `public` folder)
because here 3-4 images are okay but this would take a lot of space and time if there are many images